runtime/oti/j9nonbuilder.h

```diff
@@ -6064,7 +6064,7 @@ typedef struct J9JavaVM {
 
 #if JAVA_SPEC_VERSION >= 16
 #if JAVA_SPEC_VERSION >= 22
-#define J9_FFI_DOWNCALL_HEAP_ARGUMENT_ID 0x1
+#define J9_FFI_DOWNCALL_HEAP_ARGUMENT_ID 0xFFFFFFFFFFFFFFFFL
```

Comment:
This should use `J9CONST64` from `omrcomp.h`:
```c
#define J9_FFI_DOWNCALL_HEAP_ARGUMENT_ID J9CONST64(0xFFFFFFFFFFFFFFFF)
```

Reply:
My suggestion should not invalidate the testing underway, but may avoid compile problems on some other platforms.

Reply:
Agreed and fixed as suggested above.

Reply:
Note the macro we want to use is `J9CONST_U64`, not `J9CONST64`.

Reply:
Yes. Just fixed.